src/ex1/script.js

```diff
@@ -0,0 +1,68 @@
+// trigger fork
+let count = 0;
+
+function addTodo(){
+    const todo = document.getElementById("newTodo").value;
+    if(todo.length === 0){
+        alert("Please enter a todo task")
+    }
+    else {
+        const todoList = document.getElementById("todoList");
+        const li = document.createElement("li");
+        li.id = 'todo' + count;
+        li.innerHTML = `<input type='checkbox' name='selectTodo' class= pl value="${todo}">`;
+        li.innerHTML += todo;
+        todoList.appendChild(li);
+        document.getElementById("newTodo").value = "";
+        count++;
+        console.log("is this checked" + li.checked);
+    }
+}
+document.getElementById("addTodo").addEventListener("click", addTodo);
+
+
+let history = [];
+function deleteTodo(){
+    let todos = document.querySelectorAll('.pl');
+    const todoList = document.getElementById("todoList");
+    todos.forEach(element => {
+        console.log("checked is " + element.checked);
+        if(element.checked === true){
+            todoList.removeChild(document.getElementById(element.parentElement.id));
+            history.push(element.parentElement);
+            element.checked = false;
+        }
+    })
+}
+
+document.getElementById("delete").addEventListener("click",deleteTodo);
+
+
+// Event listeners for keypress
+window.addEventListener("keyup",e =>{
+    const todoList = document.getElementById("todoList");
+    //enable ctrl+z for adding delete items
+    if(e.ctrlKey && e.key === 'z'){
```

Comment:
Really cool addition!! 
One thing I would change here is if I deleted multiple items at one time, add them to the history as a single element (maybe push an entire array to your history array, than when you press ctrl+z it will bring all of these tasks in one time)

Reply:
Also It's a little bit annoying that if I'm inside the input (the input is focused) and I press ctrl+z it tries to bring back tasks from history..
You can check it by looking on e.target